src/app/development/input-test/input-test.component.spec.ts

```diff
@@ -0,0 +1,191 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { InputTestComponent } from './input-test.component';
+import { Component, OnInit, ViewChild } from '@angular/core';
```

Comment:
Imports should be in alphabetical order 😛

Reply:
ok, will fix this :-)

Reply:
done in https://github.com/dasch-swiss/knora-ui-ng-lib/pull/96/commits/2d8fcc115fdc796cb21faaa37e910531f87c7e65